CONTRIBUTING.md

```diff
@@ -106,7 +106,7 @@ auto:
     # This needs to be a Python-Compatible regex
     regex: 'Distribution Release: (?P<version>\d+.\d+)'
     # A template to render default value is same as in `git` above
-    # https://github.com/endoflife-date/release-data/blob/main/src/distrowatch.py
+    # https://github.com/endoflife-date/release-data/blob/main/src/distowatch.py
```

Comment:
i think we need to fix name of this py instead of making same typo here
@marcwrobel @captn3m0

Reply:
That would be ideal, but I am not familiar with how things work in https://github.com/endoflife-date/release-data so I was a bit scared to break something.

Reply:
This can be merged. The script can be renamed in other repo independently without breaking anything.

Reply:
I opened a separate issue so I can close this discussion : https://github.com/endoflife-date/release-data/issues/31.